tools/misc/calc_timeprop_maxpoldir

```diff
@@ -25,21 +25,21 @@ polarizability of that particular excitation.
 Needs mux.dat, muy.dat and muz.dat files in workind directory."""
 
 def main():
-    parser = optparse.OptionParser(usage=USAGE)
-    parser.add_option("-d", "--damping", action="store", dest="tau",
-                      help="damping constant in fs")
-    parser.add_option("-e", "--energy", action="store", dest="energy",
-                      help="excitation energy in eV")
-    parser.add_option("-w", "--wavelength", action="store", dest="wvlength",
-                      help="wavelength of excitation in nm")
-
-    (options, args) = parser.parse_args()
+    parser = argparse.ArgumentParser(usage=USAGE)
+    parser.add_argument("-d", "--damping", dest="tau",
+                        help="damping constant in fs")
```

Comment:
Same as comment below (sorry for confusing order). In my opinion the damping constant should not be optional in the first place.